Celoka/AdjustTaskSolution
# Create base app

#### What's this PR do?
- Creates a base app
- Adds a single endpoint 
#### How should this be manually tested?
- Fetch this branch, cd into its folder and create a virtual environment using the requirements file.
- Ensure your database is running locally, export the login credentials of your database to allow a successful connection
- Run the follow command:
```
python manage.py makemigrations
python manage.py migrate
python manage.py runserver
```
#### Any background context you want to provide?
- N/A
#### What are the relevant tickets?
- N/A
#### Screenshots (if appropriate)
- N/A
#### Questions:
- N/A
